build.gradle

```diff
@@ -20,7 +20,6 @@ repositories {
 dependencies {
 	developmentOnly 'org.springframework.boot:spring-boot-devtools'
 	implementation 'org.springframework.boot:spring-boot-starter-web'
-	implementation 'com.fasterxml.jackson.core:jackson-databind:2.15.2'
```

Comment:
Could you please provide some context on why this dependency was removed and what it was originally used for? Just trying to understand the rationale behind the change.